File: app/src/main/java/es/etologic/mahjongscoring2/app/utils/PreLollipopSoundPool.java
package es.etologic.mahjongscoring2.app.utils;

import android.media.AudioManager;
import android.media.SoundPool;

public final class PreLollipopSoundPool {
    @SuppressWarnings("deprecation")
    public static SoundPool NewSoundPool() {
        return new SoundPool(1, AudioManager.STREAM_MUSIC,0);
    }
}

File: app/src/main/java/es/etologic/mahjongscoring2/app/utils/PreLollipopSoundPool.kt
package es.etologic.mahjongscoring2.app.utils

import android.media.AudioManager
import android.media.SoundPool

object PreLollipopSoundPool {
    fun newSoundPool(): SoundPool {
        return SoundPool(1, AudioManager.STREAM_MUSIC, 0)
    }
}
